docs/database/azure-cosmos-db-integration.md

```diff
@@ -87,6 +87,128 @@ public class ExampleService(
 
 For more information on keyed services, see [.NET dependency injection: Keyed services](/dotnet/core/extensions/dependency-injection#keyed-services).
 
+### Add Azure Cosmos DB database
+
+<!-- TODO: Add xref to AddAzureCosmosDatabase when available -->
+
+Like container resources, you can also add a database resource. The database resource is considered a child resource to the parent <xref:Aspire.Hosting.AzureCosmosDBResource>. In your client-consuming project, you can deep-link to the database resource by name, registering a <xref:Microsoft.Azure.Cosmos.Database> instance for use with dependency injection. For example, consider the following code that calls `AddAzureCosmosDatabase` on an <xref:Microsoft.Extensions.Hosting.IHostApplicationBuilder> instance:
+
+```csharp
+builder.AddAzureCosmosDatabase(connectionName: "customers");
+```
+
+<!-- TODO: Add xref to CosmosDatabaseBuilder when available -->
+
+The `AddAzureCosmosDatabase` API returns a `CosmosDatabaseBuilder` instance that you can use to attach multiple containers under the same database connection. All child containers share the same <xref:Azure.Cosmos.CosmosClient> and database connection. This strategy is useful when associating the same <xref:Azure.Cosmos.CosmosClientOptions> with multiple containers.
```

Comment:
```suggestion
The `AddAzureCosmosDatabase` API returns a `CosmosDatabaseBuilder` instance that you can use to attach multiple containers under the same database connection. All child containers share the same <xref:Azure.Cosmos.CosmosClient> and database connection and `CosmosClient` instance. This strategy is useful when associating the same <xref:Azure.Cosmos.CosmosClientOptions> with multiple containers.
```

An important aspect of the builder is that all the containers derived from it share the same CosmosClient and the options associated with it.